### ISSUE TITLE
helleo

### ISSUE THREAD

**Issue by wxkb** [asutic/DeepISP#1]
Hello, is the code of your project Deepisp the same as the paper "DeepISP: Towards Learning an End-to-End Image Processing Pipeline"?

**Comment by asutic** [comment]
No, it is not. I've read the paper, but here the topology is inspired by U-Net. Anyway, the content in the repository is just a sketch of an idea. Not really functional.
